packages/@uppy/provider-views/src/Browser.tsx

```diff
@@ -1,16 +1,39 @@
+/* eslint-disable react/require-default-props */
```

Comment:
FYI you can circumvent this rule by replacing `prop?: T` with `prop: T | undefined`. Disabling the rule is fine though IMO.